cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # scope and that the Logging API has been enabled for the project
 # in the Google Developer Console.
 
-FROM gcr.io/google_containers/ubuntu-slim:0.4
+FROM gcr.io/google_containers/ubuntu-slim:0.4.1
```

Comment:
0.4.1 is based off https://github.com/kubernetes/contrib/blob/cd4501bdeef07db42f4856987b02f953c32885f3/images/ubuntu-slim/Dockerfile.build.
0.6 is based off https://github.com/kubernetes/ingress/blob/d002ca2f5ed08e8cf8756acae7329aa2ba536cc5/images/ubuntu-slim/Dockerfile.build

It looks like the main difference is that 0.6 doesn't delete bash. So we could probably switch to 0.6 fairly safely.
0.7 removes even more stuff, which makes it a bit riskier IMO.